src/App.tsx

```diff
@@ -11,27 +11,27 @@ interface itemsTypes {
   isEmpty: boolean;
 }
 
+
+
```

Comment:
Great Work @Jeffersonlpu ! Well done!
Just a small nitpick, it seems that your code is not correctly formatted. 
Are you using the ESLint vscode extension? If not, I highly recommend using it.

Reply:
Hi, @jhrodriguesit !
Thanks for this observation!
My ESLint was not working.
I've already updated the commit.